JASP-Engine/JASP/R/auditClassicalBenfordsLaw.R

```diff
@@ -115,16 +115,16 @@ auditClassicalBenfordsLaw <- function(jaspResults, dataset, options, ...){
     if(options[["distribution"]] == "benford"){
 
       procedureText <- base::switch(options[["digits"]],
-                                    "first" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the first leading numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel),
-                                    "firstSecond" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the first two leading numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel),
-                                    "last" = gettextf("Benford's law states that in many naturally occurring collections of numbers, the leading significant number is likely to be small. The goal of this procedure is to determine to which extent the last numbers in the population follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that do not conform to Benford's law might need further verification.", confidenceLabel))
+                                    "first" = gettextf("Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to what extent the first leading digits in the data set follow Benford's law, and to test this relation with <b>%1$s</b> confidence. Data that are not distributed according to Benford's law might need further investigation.", confidenceLabel),
```

Comment:
"Benford's law states that in many naturally occurring collections of numerical observations, the leading significant digit is likely to be small. The goal of this procedure is to determine to what extent the first leading digits in the data set follow Benford's law, and to test this relation with **a type I error of** <b>%1$s</b>. Data that are not distributed according to Benford's law might need further investigation."

So this should be 0.03 if confidence is 0.97. Confidence is not the right word here, as it relates to pre-data.